sourcecode/hub/resources/views/components/content/card.blade.php

```diff
@@ -1,16 +1,14 @@
-@props(['content', 'titlePreviews' => false])
+@props(['content'])
 
 <article class="card content-card">
     <header class="card-header content-card-header border-bottom-0 fw-bold position-relative">
         <a
-            href="{{ $content->detailsUrl }}"
+            href="{{ $content->previewUrl }}"
```

Comment:
The `href` is a fallback for non-htmx behaviour, and should be unchanged. With this change, if you right-click this link and copy the address, you'll get an internal preview URL that's useless.